src/libs/migrations/AddEncryptedAuthToken.js

```diff
@@ -0,0 +1,42 @@
+import _ from 'underscore';
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+import {reauthenticate} from '../API';
+
+const reauthenticateThrottled = _.throttle(reauthenticate, 5000);
```

Comment:
Can you add a comment explaining why this is necessary since it's not obvious (to me)

Reply:
This is gone. Was not necessary